src/compiler/crystal/tools/doc/generator.cr

```diff
@@ -164,6 +164,12 @@ class Crystal::Doc::Generator
 
   def nodoc?(str : String?)
     return false unless str
+    # TODO: keep only `:nodoc:` after 0.28.0
+    # TODO: replace puts with raise after 0.27.0
+    if str == "nodoc"
```

Comment:
`str.starts_with?("nodoc")`

Reply:
Humm... I was sure to take it during the rebase

Reply:
Go it